ddspipe_core/include/ddspipe_core/monitoring/consumers/impl/StdoutMonitorConsumer.ipp

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <iostream>
+
+// TODO:
+// The StdoutMonitorConsumer includes the producers to have access to their overloaded <<.
+// The StdoutMonitorConsumer would ideally be agonostic of the producers.
```

Comment:
No, it's not, but it can't be defined anywhere else. The consumers should be agnostic of the producers. I thought that, if they are not going to be, it's better for them to include a header with the overloaded << than to overload it themselves.

Reply:
Blindingly beautiful.